docs/JestObjectAPI.md

```diff
@@ -598,50 +598,6 @@ Returns the `jest` object for chaining.
 
 Restores all mocks back to their original value. Equivalent to calling [`.mockRestore()`](MockFunctionAPI.md#mockfnmockrestore) on every mocked function. Beware that `jest.restoreAllMocks()` only works when the mock was created with `jest.spyOn`; other mocks will require you to manually restore them.
 
-### `jest.mocked<T>(item: T, deep = false)`
```

Comment:
Just thinking out loud. Perhaps it makes sense to move this to 'TypeScript Usage' of 'Mock Functions API' docs? Feels like this way it fits better. Or?

Reply:
I think we should keep a reference here since it _is_ on `jest` object. But maybe it can just be a link to where you moved it?